sample/sample-cf.c

```diff
@@ -0,0 +1,87 @@
+//
+// Created by eugene on 3/14/19.
```

Comment:
You have a time machine? Is this the proper header?

Reply:
Should we be putting author names in files?

Reply:
> You have a time machine? Is this the proper header?

yep, I've got it in 2032